src/main/resources/assets/computercraft/lua/bios.lua

```diff
@@ -817,7 +817,8 @@ end
 settings.set( "shell.allow_startup", true )
 settings.set( "shell.allow_disk_startup", (commands == nil) )
 settings.set( "shell.autocomplete", true )
-settings.set( "edit.autocomplete", true )
```

Comment:
Wait a second why are you removing edit.autocomplete in this line? This is still a valid setting and should not be removed.